src/transactions/ExtendFootprintTTLOpFrame.cpp

```diff
@@ -154,12 +154,18 @@ ExtendFootprintTTLOpFrame::doApply(Application& app, AbstractLedgerTxn& ltx,
 
 bool
 ExtendFootprintTTLOpFrame::doCheckValid(SorobanNetworkConfig const& config,
```

Comment:
Nit: change `config` to `networkConfig` or `sorobanConfig` for clarity, both here and in the other ops that now take both a network config and app config as parameters.

Reply:
Done.